src/model/HeatPumpPlantLoopEIRCooling.cpp

```diff
@@ -0,0 +1,491 @@
+    capacityModifierFunctionofTemperatureCurve.setCoefficient5yPOW2(0);
+    capacityModifierFunctionofTemperatureCurve.setCoefficient6xTIMESY(0);
+    capacityModifierFunctionofTemperatureCurve.setMinimumValueofx(5);
+    capacityModifierFunctionofTemperatureCurve.setMaximumValueofx(10);
+    capacityModifierFunctionofTemperatureCurve.setMinimumValueofy(24);
+    capacityModifierFunctionofTemperatureCurve.setMaximumValueofy(35);
+    capacityModifierFunctionofTemperatureCurve.setInputUnitTypeforX("Temperature");
+    capacityModifierFunctionofTemperatureCurve.setInputUnitTypeforY("Temperature");
+    bool ok = setCapacityModifierFunctionofTemperatureCurve(capacityModifierFunctionofTemperatureCurve);
+    OS_ASSERT(ok);
+
+    CurveBiquadratic electricInputtoOutputRatioModifierFunctionofTemperatureCurve(model);  // PlantLoopHeatPump_EIR_WaterSource.idf
+    electricInputtoOutputRatioModifierFunctionofTemperatureCurve.setCoefficient1Constant(1);
+    electricInputtoOutputRatioModifierFunctionofTemperatureCurve.setCoefficient2x(0);
+    electricInputtoOutputRatioModifierFunctionofTemperatureCurve.setCoefficient3xPOW2(0);
+    electricInputtoOutputRatioModifierFunctionofTemperatureCurve.setCoefficient4y(0);
+    electricInputtoOutputRatioModifierFunctionofTemperatureCurve.setCoefficient5yPOW2(0);
+    electricInputtoOutputRatioModifierFunctionofTemperatureCurve.setCoefficient6xTIMESY(0);
+    electricInputtoOutputRatioModifierFunctionofTemperatureCurve.setMinimumValueofx(5);
+    electricInputtoOutputRatioModifierFunctionofTemperatureCurve.setMaximumValueofx(10);
+    electricInputtoOutputRatioModifierFunctionofTemperatureCurve.setMinimumValueofy(24);
+    electricInputtoOutputRatioModifierFunctionofTemperatureCurve.setMaximumValueofy(35);
+    electricInputtoOutputRatioModifierFunctionofTemperatureCurve.setInputUnitTypeforX("Temperature");
+    electricInputtoOutputRatioModifierFunctionofTemperatureCurve.setInputUnitTypeforY("Temperature");
+    ok = setElectricInputtoOutputRatioModifierFunctionofTemperatureCurve(electricInputtoOutputRatioModifierFunctionofTemperatureCurve);
+    OS_ASSERT(ok);
+
+    CurveQuadratic electricInputtoOutputRatioModifierFunctionofPartLoadRatioCurve(model);  // PlantLoopHeatPump_EIR_WaterSource.idf
+    electricInputtoOutputRatioModifierFunctionofPartLoadRatioCurve.setCoefficient1Constant(1);
+    electricInputtoOutputRatioModifierFunctionofPartLoadRatioCurve.setCoefficient2x(0);
+    electricInputtoOutputRatioModifierFunctionofPartLoadRatioCurve.setCoefficient3xPOW2(0);
+    electricInputtoOutputRatioModifierFunctionofPartLoadRatioCurve.setMinimumValueofx(0);
+    electricInputtoOutputRatioModifierFunctionofPartLoadRatioCurve.setMaximumValueofx(1);
+    ok = setElectricInputtoOutputRatioModifierFunctionofPartLoadRatioCurve(electricInputtoOutputRatioModifierFunctionofPartLoadRatioCurve);
+    OS_ASSERT(ok);
+
+    setCondenserType("WaterSource");
+    setReferenceCoefficientofPerformance(7.5);
```

Comment:
E+ example has 3.5 not 7.5. But 7.5  is the IDD default for `<insert reason>`
```suggestion
    setReferenceCoefficientofPerformance(7.5);  // IDD default
```